Character.h

```diff
@@ -10,65 +10,73 @@
 struct Character
 {
     Character(int hp, int armor_, int attackDamage_ );
+
     virtual ~Character() { }
     
     /*
-     a pure virtual getName function.
-     derived class stores the name, not the base class.
-     */
+    a pure virtual getName function.
+    derived class stores the name, not the base class.
+    */
     
     virtual const std::string& getName() = 0;
+
     virtual std::string getStats() = 0;
     
     virtual void attack( Character& other );
+
     void defend();
     
     void help( Character& other );
     
     int takeDamage(int damage);
     
-    int getHP() const { return hitPoints; }
-    int getArmorLevel() const { return armor; }
-    int getAttackDamage() const { return attackDamage; }
-    bool getIsDefending() const { return isDefending; }
-    
-    const std::vector<std::unique_ptr<Item>>& getHelpfulItems() const { return helpfulItems; }
-    const std::vector<std::unique_ptr<Item>>& getDefensiveItems() const { return defensiveItems; }
+    int getHP() const;
+
+    int getArmorLevel() const;
+
+    int getAttackDamage() const;
+
+    bool getIsDefending() const;
```

Comment:
You did not need to revise any of these.  Put them back the way they were.
- [ ] revise.

Reply:
done